src/crypto-api/index.ts

```diff
@@ -364,6 +365,22 @@ export interface CryptoApi {
      */
     getEncryptionInfoForEvent(event: MatrixEvent): Promise<EventEncryptionInfo | null>;
 
+    /**
+     * Encrypts a given payload object via Olm to-device messages to a given
+     * set of devices.
+     *
+     * @param eventType the type of the event to send
+     * @param devices an array of (user ID, device ID) pairs to encrypt the payload for
```

Comment:
"pair" implies a 2-tuple to me, which isn't what these are. We can probably just gloss over it and let the type speak for itself:

```suggestion
     * @param devices - an array of devices to encrypt the payload for.
```